boards/PDM/Src/Io/Io_CurrentSense.c

```diff
@@ -1,10 +1,16 @@
 #include "Io_CurrentSense.h"
+#include "Io_Adc.h"
+#include "main.h"
 
 // TODO: Fix hard-coded values once the ADC is configured to read load-switches
 
 float Io_CurrentSense_GetAux1Current(void)
 {
-    return 0.5f;
+    // Aux1 Current = ADC Voltage * Current Gain Ratio
+
+    const float LOW_CURRENT_SENSE_GAIN_RATIO = 500.0f;
+
+    return (Io_Adc_GetChannel6Voltage() * LOW_CURRENT_SENSE_GAIN_RATIO);
```

Comment:
```suggestion
    const float LOW_CURRENT_SENSE_GAIN_RATIO = 500.0f;

    // Aux1 Current = ADC Voltage * Current Gain Ratio
    return (Io_Adc_GetChannel6Voltage() * LOW_CURRENT_SENSE_GAIN_RATIO);
```

Reply:
Copied the same order you had used
https://github.com/UBCFormulaElectric/Consolidated-Firmware/blob/master/boards/FSM/Src/Io/Io_SteeringAngleSensor.c

Reply:
This could be a consistency moment